system-variables.md

```diff
@@ -5230,7 +5230,14 @@ For details, see [Identify Slow Queries](/identify-slow-queries.md).
 - Type: Integer
 - Default value: `4096`
 - Range: `[0, 2147483647]`
-- This variable is used to control the length of the SQL string in [statement summary tables](/statement-summary-tables.md).
+
+<CustomContent platform="tidb">
+- This variable is used to control the length of the SQL string in [statement summary tables](/statement-summary-tables.md), the [SLOW_QUERY](/information-schema/information-schema-slow-query.md) table and on the [TiDB Dashboard](/dashboard/dashboard-intro.md).
```

Comment:
What's on the dashboard? The SLOW_QUERY?

Reply:
@hfxsd Did you see the example in my comment? https://github.com/pingcap/docs/pull/15602#issuecomment-1841156401

Reply:
The dashboard likely uses the `SLOW_QUERY` table but the user might not be aware of that.